### ISSUE TITLE
getOption, to get values of other VM parameters

### ISSUE THREAD

**Issue by RagingLink** [ChewbaccaCookie/voicemeeter-connector#34]
* **I'm submitting a ...**

- [ ] bug report
- [x] feature request
- [ ] question about the decisions made in the repository
- [ ] question about how to use this project

* **Summary**
A way to get values of parameters you can set with `.setOption()`, e.g: 
```js
if (vm.getOption('recorder.Pause') === 0) vm.setOption('recorder.Pause=1')
```

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)
I figured it'd be possible to use code from `getParameter`, since `setOption()` also utilized code of `setParameter` [(#31) feat: Add a way to set options for voicemeeter  ](https://github.com/ChewbaccaCookie/voicemeeter-connector/pull/31), but I can't get **ffi-napi** to build at all so I can't just test out the change easily

**EDIT:** It totally works when borrowing code from getParameter, I'm just not too familiar with Voicemeeter and its 'API' so I'm really not confident in making a pull request

**Comment by ChewbaccaCookie** [comment]
I like the idea, you already created a PR for this. We can close the issue after the PR was merged and a release created :)

**Comment by ChewbaccaCookie** [comment]
This feature is now released in v1.3.0 🚀
